resources/views/layouts/master.blade.php

```diff
@@ -71,6 +71,10 @@
                 <li {{ (Request::is('online-donations*') ? 'class=active' : '') }}><a href="/online-donations">Online Donations </a></li>
                 <li {{ (Request::is('twitter-feed*') ? 'class=active' : '') }}><a href="/twitter-feed">#FloodSL Twitter Feed</a></li>
             </ul>
+            <ul class="nav navbar-nav navbar-right">
+                <li {{ (App::isLocale('si') ? 'class=active' : '') }}><a href="/locale/si">සිංහල</a></li>
+                <!--<li {{ (App::isLocale('ta') ? 'class=active' : '') }}><a href="/locale/ta">தமிழ்</a></li>-->
```

Comment:
This is commented out since we need to have the translations in place to activate it. Since the mechanics are in place its now only a matter of getting the content.